src/tests/inventorySorting.spec.js

```diff
@@ -0,0 +1,67 @@
+import { expect } from '@playwright/test';
+import { test } from '../fixtures/base';
+
+test.describe('Perform and verify sorting on the Inventory page', () => {
+    test('Sorting products name A-Z', async (
+        /** @type {{ app: import('../pages/Application').Application }} */{ app },
+    ) => {
+        await app.login.navigate();
+        await app.login.performLogin('standard_user', 'secret_sauce');
+
+        expect(await app.inventory.inventoryItems.count()).toBeGreaterThanOrEqual(1);
+
+        const productNamesBeforeSort = await app.inventory.inventoryItems.allTextContents();
+
+        await app.inventory.sortProductNameAZ();
+        const productNamesAfterSortAZ = await app.inventory.inventoryItems.allTextContents();
+        const sortedNamesAZ = [...productNamesBeforeSort].sort();
+        expect(sortedNamesAZ).toEqual(productNamesAfterSortAZ);
```

Comment:
You can also enable an additional check to make sure the array is not empty before sorting